akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -834,25 +837,38 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
 
   // InboundContext
   override def sendControl(to: Address, message: ControlMessage) =
-    association(to).sendControl(message)
-
-  override def send(message: Any, sender: OptionVal[ActorRef], recipient: RemoteActorRef): Unit = {
-    val cached = recipient.cachedAssociation
+    try {
+      association(to).sendControl(message)
```

Comment:
I see
It can also be caused by `LazyQueueWrapper` which indirectly can call the run*Stream method
Looks like we need the silence at both places to be sure.